stan/math/prim/mat/err/check_spsd_matrix.hpp

```diff
@@ -29,9 +28,11 @@ namespace stan {
      */
     template <typename T_y>
     inline bool
-    check_spsd_matrix(const char* function,
-                      const char* name,
-                      const Eigen::Matrix<T_y, Dynamic, Dynamic>& y) {
+    check_spsd_matrix(
+      const char* function,
+      const char* name,
+      const Eigen::Matrix<T_y, Eigen::Dynamic, Eigen::Dynamic>& y
+    ) {
```

Comment:
I guess there's a lot of them.  I'll let you find the rest if you don't mind changing them.  If you don't want to change them, it's no big deal.  Maybe other people don't agree, but I find the line that starts with a close paren odd and hard to parse.